src/commands/classes/Bot.prep.ts

```diff
@@ -0,0 +1,14 @@
+import fs from "fs";
```

Comment:
I'm not sure the purpose of this file. Using a writeSync with the 'w' flag will auto-create files that don't exist. Is there a reason you've abstracted that out to this file?